.travis.yml

```diff
@@ -7,11 +7,18 @@ cache:
   directories:
   - node_modules
 before_install:
-  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.7.0
```

Comment:
Maybe we can just the yarn from Travis, now it's correctly updated with the last version.